incuna-sass/mixins/_em-scale.sass

```diff
@@ -26,14 +27,14 @@
 
 // You can also use variables to reference media query variables in your
 // project:
-//     @include em-scale(14px, (($px768, 16px), ($px1025, 20px)))
+//     @include em-scale(14px, ($px768: 16px, $px1025: 20px))
 // or
-//     @include em-scale(14px, (($tablet, 16px), ($desktop, 20px)))
+//     @include em-scale(14px, ($tablet: 16px, $desktop: 20px))
 
-@mixin em-scale($font-base, $breakpoints-sizes-list: ())
+@mixin em-scale($font-base, $breakpoints-map)
     font:
         size: $font-base
-    @each $change in $breakpoints-sizes-list
-        @include media(#{nth($change, 1)})
+    @each $breakpoint, $font-size in $breakpoints-map
```

Comment:
Cool! That works nicely because Sass map keys can be anything:
>Both the keys and values in maps can be any SassScript object.
http://www.sass-lang.com/documentation/file.SASS_REFERENCE.html#maps

@hippogriffic Can you recommend in the examples to use variables for the map keys, for readability? Keep the example without variables because it explains what's going on quite well, but a recommendation would be nice.